src/macros.jl

```diff
@@ -416,6 +416,7 @@ macro wrap(ex)
 
     return esc(MacroTools.@qq begin
        @eval const $newsym = Ref{GapObj}()
+       global $name;
```

Comment:
Any deep reason why a semicolon is needed? If yes it should be documented in a comment. If not, let's drop it
```suggestion
       global $name
```